_posts/2025-03-13-CPANSec-Retrospective-2024.md

```diff
@@ -0,0 +1,146 @@
+But CPANSec follow closely and contribute to them when possible.
+
+## CPANSec as a CVE Numbering Authority (CNA)
+In 2023 and 2024, CPANSec helped with or contributed CVEs for Perl, 
+but assignation remained under control of another CVE Numbering Authority (CNA).
+
+CPANSec worked through the year to get more empowerment and ability to assign CVEs.
+After all, who knows better than CPANSec the impact of CVEs on Perl modules :)
+
+And finally, by end of 2024, CPANSec has laid the groundwork to become a CNA.
+
+This designation was officially granted in February 2025: [CPANSec is now a CVE Numbering Authority (CNA)!](https://security.metacpan.org/2025/02/25/cpansec-is-cna-for-perl-and-cpan.html)
+
+The news of CPANSec becoming CVE Numbering Authority (CNA) was saluted by big names like Greg Kroah-Hartman (GNU/Linux) and Daniel Stenberg (curl).
+
+## Introduced a SECURITY Policy
+In parallel, CPANSec pushed for authors to publish a Security Policy. 
+
+Doing so is part of Open Source good practices and security becoming more and more critical.
+
+This effort resulted in creating a [template](https://security.metacpan.org/docs/guides/security-policy-for-authors.html#examples), providing [guidelines](https://security.metacpan.org/docs/guides/security-policy-for-authors.html) and, last but not least, communicating about the initiative (various channels).
+
+## Study on Random Number generation for Security
+Generating random numbers can help in a wide variety of tasks.
+All of them don't have the same level of criticality.
+And in particular, random generated data used in a security context requires extra care.
+
+CPANSec studied and reviewed for you the CPAN modules for this usage in a document [Random Data For Security Use](https://security.metacpan.org/docs/guides/random-data-for-security.html).
+
+## Working on "TLS in core"
+A vanilla install of Perl (distribution) does not provide HTTPs capabilities, and this is annoying.
+That's an important (and difficult) topic that is addressed, across core developpers and CPANSec.
+
+Things started to go into motion in 2024 to make it happen.
+
+[Discussions](https://www.nntp.perl.org/group/perl.perl5.porters/2024/11/msg269050.html), with the Perl Steering Council or core contributor. A [plan](https://www.nntp.perl.org/group/perl.perl5.porters/2024/11/msg269202.html) was built, with some preliminary work happening.
+
+The creation of the new module [Crypt::Bear](https://metacpan.org/dist/Crypt-Bear) to wrap the TLS library BearSSL.
+
+## SBOM
+CPANSec attended a billion of meetings, cross connected with several other organization working on the topic. 
+
+One of the outcome of this huge effort is available as a document [Roles and metadata in open source supply-chains](https://security.metacpan.org/docs/supplychain-sbom.html) but it goes well beyond that.
+
+## Adopting security related modules
+CPANSec adopted security related modules:
```

Comment:
s/CPANSec /CPANSec volunteers /; … and list their names! I think these types of contributions shouldn't go uncredited